libs/@blockprotocol/service/src/react.ts

```diff
@@ -0,0 +1,46 @@
+import { useServiceConstructor } from "@blockprotocol/core/react";
+import { RefObject } from "react";
+
+import { ServiceBlockHandler, ServiceEmbedderHandler } from "./index.js";
+
+/**
+ * Create a ServiceBlockHandler instance, using a reference to an element in the block.
+ *
+ * The serviceModule will only be reconstructed if the element reference changes.
+ * Updates to any callbacks after first constructing should be made by calling serviceModule.on("messageName", callback);
+ */
+export const useServiceBlockService = (
```

Comment:
```suggestion
export const useServiceBlockModule = (
```

Reply:
Addressed in https://github.com/blockprotocol/blockprotocol/commit/32a3ca931e9f5c2229fcb9075b25a062d828b671